.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: Report a bug in freud.
+labels: ['bug']
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Please provide a *minimal* script to reproduce the bug.
+      render: python
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Error Output
+      description: Please indicate the output from the above script and what you expect the output to be.
+      render: shell
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: freud Version
+      description: What version of freud are you using?
+      placeholder: v2.11.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Python Version
+      description: What version of python are you using?
+      placeholder: v3.8.2
+  - type: dropdown
+    attributes:
+      label: System Platform
+      description: What platform are you using?
+      options:
+        - MacOS
+        - Linux
+        - Windows
+  - type: dropdown
+    attributes:
+      label: Installation method
+      description: How did you install freud?
+      options:
+        - Download from PyPI
+        - Download from conda-forge
+        - glotzerlab-software container
+        - built from source
+  - type: dropdown
+    attributes:
+      label: Developer
+      description: Who should fix this?
+      options:
+        - I will submit a pull request that fixes this problem.
+        - Would someone else please fix this?
+      validations:
+        required: true
```

Comment:
```suggestion
```
On second though, let's remove this section. Users that plan to implement a fix can mention so in the comments.

Reply:
How about we just remove the required validation?